### PR TITLE
[pt-br] Update translation of Names and UIDs page

### PR DESCRIPTION
**Pull request by stormqueen1990**
### Changes

* Update the translation of the Names and UIDs page and update all references to use the new pt-br version of the DNS subdomain names in the Brazilian Portuguese version of documentation.

### CODE REVIEW

File: content/pt-br/docs/reference/glossary/name.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Name
```
Comment (sftim) [inline content/pt-br/docs/reference/glossary/name.md:2]:
```suggestion
title: Nome
```
?

**Review by Arhell — COMMENTED** [review]
/assign @edsoncelio

File: content/pt-br/docs/reference/glossary/name.md (hunk 1 above)
Comment (stormqueen1990) [reply content/pt-br/docs/reference/glossary/name.md]: Good catch, thanks! Fixed!

**Comment by rikatz** [comment]
Thanks!

**Pull request merged**